core/src/main/groovy/noe/tomcat/configure/ConnectorSSLHostConfigTomcat.groovy

```diff
@@ -0,0 +1,110 @@
+package noe.tomcat.configure
+
+/**
+ * Abstraction for SSLHostConfig tag of either Secure or Non-Secure Connector in Tomcat server.xml.
```

Comment:
The term 'element' is used across the code (instead of tag).

Reply:
Nice suggestion. Missed it :) Will replace it asap